### ISSUE TITLE
Line break behavior is different from xargs

### ISSUE THREAD

**Issue by robario** [greymd/tmux-xpanes#52]
```
$ echo -ne "1\n\n2\n\n" | xargs printf '[%s]'
[1][2]
```

```
$ echo -ne "1\n\n2\n\n" | xpanes printf '[%s]'

# will open 4 panes, expected 2 panes
```

**Comment by greymd** [comment]
It is not difficult to fix this issue soon, but it will most likely be resolved with another issue (#24) at the same time.
As the design direction of tmux-xpanes, I will try to implement the tokenization logic same as `xargs` as possible (It might be version 2.1.0).

Stay tuned :)

(Or, welcome any pull requests.)

**Comment by greymd** [comment]
memo
It seems that xargs skips some characters in IFS.

```sh
$ echo -n "$IFS" | od -tx1c
0000000  20  09  0a  00
             \t  \n  \0
0000004
```

Not only line breaks, but also space and tab.

```sh
$ echo -ne "aa\n\n\t  \t\n\nbb" | xargs printf "[%s]"
[aa][bb]
```

NUL character causes warning message.

```sh
$ echo -ne "aa\n\n\t  \t\0\n\nbb" | xargs printf "[%s]"
xargs: WARNING: a NUL character occurred in the input.  It cannot be passed through in the argument list.  Did you mean to use the --null option?
[aa][][bb]% 

$ echo $?
0
```

**Comment by greymd** [comment]
Fixed with [v2.1.0](https://github.com/greymd/tmux-xpanes/releases/tag/v2.1.0).

```
$ echo -ne "1\n\n2\n\n" | xargs printf '[%s]'
```

Now will generate

```
root@e08434b0b7ba:/# printf [%s] 1             |root@e08434b0b7ba:/# printf [%s] 2                   
[1]root@e08434b0b7ba:/#                        |[2]root@e08434b0b7ba:/#                              
                                               |                                                     
                                               |                                                     
                                               |                                                     
                                               |                                                     
                                               |                                                     
                                               |                                                     
                                               |                                                     
                                               |                                                     
                                               |                                                     
                                               |                                                     
                                               |                                                     
                                               |                                                     
                                               |                                                     
```

@robario plz check it out.

**Comment by robario** [comment]
👍 perfect! @greymd
